### PR TITLE
fix deprecation warnings related to Int128

### PR DESCRIPTION
**Pull request by narimiran**

### CODE REVIEW

File: lib/system/io.nim [hunk 1]
```diff
@@ -666,7 +666,7 @@ proc readFile*(filename: string): TaintedString {.tags: [ReadIOEffect], benign.}
   var f: File
   if open(f, filename):
     try:
-      result = readAll(f).TaintedString
```
Comment (Araq) [inline lib/system/io.nim]: Unrelated commit that could cause harm.

Comment (Clyybber) [reply lib/system/io.nim]: I don't think so. `readAll` already returns a TaintedString.

Comment (narimiran) [reply lib/system/io.nim]: Exactly what @Clyybber says. Compiler even warns that the original code is a pointless conversion to itself.
